### ISSUE TITLE
Theme source for nativeTheme on Linux

### ISSUE THREAD

**Issue by zarubond** [electron/electron#21325]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
7
* **Operating System:**
  Ubuntu 18.04
* **Last Known Working Electron version:**
never, new feature

### Expected Behavior
Settings themeSource on nativeTheme to 'dark/light' should change color of context menus accordingly.

### Actual Behavior
The color of context menus it always the system one. 

### To Reproduce
Set your OS color to some black and then set themeSource to light. 

### Screenshots
![Screenshot_20191128_143056](https://user-images.githubusercontent.com/6961047/69810558-7c161f00-11ec-11ea-8326-2d5e83fa9966.png)

**Comment by dbjorge** [comment]
In a default Ubuntu 18.04.3 LTS (Gnome) install, querying `dconf` looks like a reasonable starting point: 

![image](https://user-images.githubusercontent.com/376284/74996275-01822780-5408-11ea-8510-186a71d40871.png)

```bash
> dconf read /org/gnome/desktop/wm/preferences/theme
'HighContrast'
```

**Comment by MarshallOfSound** [comment]
@zarubond I think this was fixed as a side-effect of https://github.com/electron/electron/pull/22896 can you test again on a recent version of Electron 8 or 9?

**Comment by zarubond** [comment]
This will be hopefully fixed by #23678

**Comment by MarshallOfSound** [comment]
Thank you for reaching out!

The version of Electron reported in this issue has reached end-of-life and is [no longer supported](https://www.electronjs.org/docs/tutorial/support#supported-versions), so this issue will be closed.

If you're still experiencing this issue on a [supported version](https://www.electronjs.org/releases/stable) of Electron, please file a new issue for that version of Electron.

Thanks in advance! Your help is appreciated.
